frappe/integrations/doctype/s3_backup_settings/s3_backup_settings.py

```diff
@@ -106,8 +106,8 @@ def send_email(success, service_name, error_status=None):
 	if not frappe.db:
 		frappe.connect()
 
-	if frappe.db.get_value("S3 Backup Settings", None, "notification_email"):
-		recipients = split_emails(frappe.db.get_value("S3 Backup Settings", None, "notification_email"))
+	if frappe.db.get_value("S3 Backup Settings", None, "notify_email"):
```

Comment:
Hi, while you're at this could we assign this to a variable and then check for the value in the variable instead of calling `frappe.db.get_value()` twice?

something like this should do:
```python3
	notify_email = frappe.db.get_value("S3 Backup Settings", None, "notify_email")
	if notify_email:
		recipients = split_emails(notify_email)
	...
```

Reply:
Good idea. I will make changes accordingly to your suggestion. Thank you.

Reply:
yeah, good catch, we can skip the if condition.